lib/katello/engine.rb

```diff
@@ -85,15 +85,6 @@ class Engine < ::Rails::Engine
       end
     end
 
-    initializer "katello.initialize_cp_listener", :before => :finisher_hook do
```

Comment:
I'm thinking we could run both of the event monitors here just like in the rake task for dev?

Reply:
makes sense, as long as it doesn't cause any issues :+1:

Reply:
I think I resolved the daemon running in the spring process by checking $PROGRAM_NAME in EventDaemon!